atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/KvsLockWatchingService.java

```diff
@@ -16,34 +16,25 @@
 
 package com.palantir.atlasdb.transaction.api;
 
-import java.util.Set;
+import java.util.OptionalLong;
 
-import com.palantir.atlasdb.keyvalue.api.TableReference;
 import com.palantir.common.annotation.Idempotent;
-import com.palantir.lock.watch.LockWatchState;
-import com.palantir.lock.watch.WatchId;
+import com.palantir.lock.watch.LockWatchRequest;
 
 public interface KvsLockWatchingService {
     /**
-     * Registers watches for a set of rows in a single table.
-     * @param tableRef table to register watches for.
-     * @param rowNames rows to register watches for.
-     * @return a mapping of {@link WatchId}s to the {@link RowReference} for which the watches have been registered
+     * Registers watches.
```

Comment:
at this point, there's no KvsLockWatchingService, right? shoudl this PR delete that class?